react/features/jaas/middleware.web.js

```diff
@@ -1,25 +1,39 @@
 import { redirectToStaticPage } from '../app/actions';
 import { CONFERENCE_JOINED } from '../base/conference/actionTypes';
+import { CONNECTION_FAILED } from '../base/connection';
+import { JitsiConnectionErrors } from '../base/lib-jitsi-meet';
 import { MiddlewareRegistry } from '../base/redux';
 
 import { SET_DETAILS } from './actionTypes';
 import { getCustomerDetails } from './actions';
 import { STATUSES } from './constants';
 
+const redirectErrors = [
+    JitsiConnectionErrors.CONNECTION_DROPPED_ERROR,
+    JitsiConnectionErrors.PASSWORD_REQUIRED
```

Comment:
Isn't this also used for prompting the password dialog in case a meeting is pass protected? Do we want to redirect in that case too?